CHANGELOG.adoc

```diff
@@ -60,7 +60,7 @@ The changes are:
 - https://github.com/eclipse-syson/syson/issues/1012[#1012] [general-view] Allow `Package` and `NamespaceImport` graphical nodes to be smaller than their default size
 - https://github.com/eclipse-syson/syson/issues/1006[#1006] [details] Display `FeatureValue.isDefault` and `FeatureValue.isInitial` in Core tab of `FeatureValue` concept.
 - https://github.com/eclipse-syson/syson/issues/1018[#1018] [libraries] Make customizing the default metamodels and libraries available in SysML projects easier by making default implementation `SysONDefaultLibrariesConfiguration` more extensible.
-
+- https://github.com/eclipse-syson/syson/issues/1033[#1033] [common-view] Make it possible to create a diagram representation directly under a root namespace element
```

Comment:
In the release notes, you are talking about the General View, here you seem to talk about all diagrams. Which one is correct?

Also please move it to 2025.4.0 section